packages/community-cli-plugin/src/commands/bundle/buildBundle.js

```diff
@@ -99,6 +100,7 @@ async function buildBundleWithConfig(
     minify: args.minify !== undefined ? args.minify : !args.dev,
     platform: args.platform,
     unstable_transformProfile: args.unstableTransformProfile,
+    customResolverOptions: args.resolverOptions,
```

Comment:
I'd name it the same as the Metro's flag to avoid ambiguity

Reply:
👍